task0/Python_Intro_Ferhat.ipynb

```diff
@@ -665,16 +665,35 @@
   },
```

Comment:
<p>Be more explicit in your variable naming:</p><p> </p><pre class="ql-syntax" spellcheck="false">total_sum = 0

for i in range(10,21):
&nbsp; &nbsp; if i % 2 == 1:
&nbsp; &nbsp; &nbsp; &nbsp; total_sum += i

print('Sum of the numbers: ', total_sum)&nbsp;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>